apache/superset
# chore: Remove crud.less from Datasource

### SUMMARY
Removes the crud.less file from the Datasource component

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
No visual changes

### TESTING INSTRUCTIONS
1. Go to Explore
2. Open the Edit dataset modal
3. Make sure there are no visual changes

### ADDITIONAL INFORMATION
